lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -17,7 +17,8 @@ if Enum.all?(
     ]
     ```
 
-    It can perform transcoding, in such case these plugins are also needed:
+    It can perform transcoding (more information in `Membrane.RTC.Engine.Endpoint.HLS.TranscodingConfig` moduledoc),
+    in such case these plugins are also needed:
```

Comment:
```suggestion
    It can perform transcoding (see `Membrane.RTC.Engine.Endpoint.HLS.TranscodingConfig`),
    in such case these plugins are also needed: